pkg/agent/openflow/pipeline.go

```diff
@@ -2106,8 +2063,11 @@ func (f *featureNetworkPolicy) dnsPacketInFlow(conjunctionID uint32) binding.Flo
 	return AntreaPolicyIngressRuleTable.ofTable.BuildFlow(priorityDNSIntercept).
 		Cookie(f.cookieAllocator.Request(f.category).Raw()).
 		MatchConjID(conjunctionID).
-		Action().LoadToRegField(CustomReasonField, CustomReasonDNS).
-		Action().SendToController(uint8(PacketInReasonNP)).
+		// FQDN should pause DNS response packets and send them to the controller. After
```

Comment:
Could you explain more about `pause`? maybe better to add a few comments before method SendToController for this new parameter.

Reply:
Added.